bindings/go/oci/ctf/store_test.go

```diff
@@ -378,3 +378,34 @@ func TestResolveWithRegistry(t *testing.T) {
 		assert.Equal(t, digest.Digest(digestStr), desc.Digest)
 	})
 }
+
+func TestResolveWithEmptyMediaType(t *testing.T) {
```

Comment:
You should also test integration with an old CTF that you open

Reply:
What would that change? Wouldn't it be the same in regards to the mediatype just being empty? 🤔